.github/workflows/cmake.yml

```diff
@@ -254,6 +255,7 @@ jobs:
            compiler: {c: clang-cl, cxx: clang-cl}
          - adapter: {name: L0, var: '-DUR_BUILD_ADAPTER_L0=ON -DUR_STATIC_ADAPTER_L0=ON'}
            compiler: {c: clang-cl, cxx: clang-cl}
+         - compiler: {c: clang-cl, cxx: clang-cl}
```

Comment:
might be easier to just comment out clangs from the compiler list below.

Reply:
Have changed it to that.